lib/v2/jou/utils/index.js

```diff
@@ -0,0 +1,67 @@
+// 导入got库，该库用来请求网页数据
+const got = require('@/utils/got');
+// 导入cheerio库，该库用来解析网页数据
+const cheerio = require('cheerio');
+// 导入parseDate函数，该函数用于日期处理
+const { parseDate } = require('@/utils/parse-date');
+// 导入timezone库，该库用于时区处理
+const timezone = require('@/utils/timezone');
+
+async function getItems(ctx, url, host, tableClass, timeStyleClass, titleStyleClass) {
+    // 发起Http请求，获取网页数据
+    const response = await got({ url, https: { rejectUnauthorized: false } });
+    // 解析网页数据
+    const $ = cheerio.load(response.data);
+
+    // 通知公告的items的标题、url链接、发布日期
+    const list = $(`table.${tableClass} > tbody > tr`)
+        .filter((index, item) => {
+            const currentItem = $(item);
+            const item1 = currentItem.find('td:eq(1)');
+            const hasLink = item1.find('a').length > 0;
+            const isLastItem = index === $(`table.${tableClass} > tbody > tr`).length - 1;
+            return hasLink && !isLastItem;
+        })
+        .toArray()
```

Comment:
If you want to remove the last item from an array, you can use 
```js
.slice(0, -1)
```

You can also filter at the selector level
```suggestion
    const list = $(`table.${tableClass} > tbody > tr[height=20]`)
        .toArray()
```